gallery/src/styles/NxCard/NxCardColumnLayoutExample.tsx

```diff
@@ -12,7 +12,7 @@ import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
 export default function NxCardColumnLayoutExample() {
   return (
     <div className="nx-card-container nx-card-container--column">
-      <section className="nx-card">
+      <section className="nx-card" aria-label="Card 1 in Column Layout">
```

Comment:
I've noticed most of the aria-label examples I've seen seem to use only lower-case. I'm guessing it's just because the screen readers don't pay attention to the capitalization anyway and laziness.